test/unit/service/event-service.ts

```diff
@@ -310,10 +311,10 @@ describe('eventService', () => {
         startDate: event.startDate,
       });
     });
-    it('should not send a reminder to events starting in three days and one millisecond from now', async () => {
+    it('should not send a reminder to events starting in three days and one minute from now', async () => {
```

Comment:
The timestamp you create below is 4 days and one second from now, not three days and one minute

Reply:
Still a second, not a minute :\